packages/web/src/store/slices/websiteResults.ts

```diff
@@ -0,0 +1,75 @@
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import type { WebPage, WebPagePackage, PackageVulnerabilityData } from '@gradejs-public/shared';
+import { DefaultFiltersAndSorters, FiltersState } from '../../components/layouts/Filters/Filters';
+import { client } from '../../services/apiClient';
+import { trackCustomEvent } from '../../services/analytics';
+
+const defaultDetectionResult = {
+  packages: [] as WebPagePackage[],
+  vulnerabilities: {} as Record<string, PackageVulnerabilityData[]>,
+  webpages: [] as WebPage[],
+};
+
+const initialState = {
+  filters: { ...DefaultFiltersAndSorters },
```

Comment:
Why can't we have filters state on the component level? I believe we should reset them on every website

Reply:
This is a controversial point, there use cases that would benefit from non-resetting filters, we might even want to persist filter state in url query. Let's review this later, during redesign.

Reply:
Filtering is not a UI thing, it's data related thing. It makes much more sense to keep it in the app state.